src/scss/layout/_footer.scss

```diff
@@ -2,6 +2,89 @@
 
 [data-layout='footer'] {
   @include tools.font-family('alt');
+  display: grid;
   font-size: var(--small);
   grid-area: footer;
+  grid-gap: var(--gutter);
+  padding: var(--gutter) 0 var(--spacer);
+
+  @media (min-width: 45em) {
```

Comment:
I don't know - I was curious about that too, trying to decide. So far I haven't used `before()`/`after()` either, because I wasn't sure they provide much use. But `above()`/`below()` allow us to re-use some common sizes, which might be good…